supabase/functions/sync-sprint-to-notion/slack.ts

```
@@ -0,0 +1,18 @@
+const SLACK_API = 'https://slack.com/api';
+
+export async function slackPostToChannel(token: string, channel: string, text: string): Promise<void> {
```

ESLint

[error] Replace `token:·string,·channel:·string,·text:·string` with `⏎··token:·string,⏎··channel:·string,⏎··text:·string,⏎`

(prettier/prettier)